gearadmin/gearadmin.go

```diff
@@ -85,3 +133,16 @@ func (ga GearmanAdmin) Workers() ([]Worker, error) {
 	}
 	return workers, scanner.Err()
 }
+
+func (ga GearmanAdmin) Cancel(handle string) (bool, error) {
+	fmt.Fprintf(ga.conn, fmt.Sprintf("cancel %s\n", handle))
```

Comment:
I think it was `cancel-job`.
https://github.com/gearman/gearmand/blob/master/bin/gearadmin.cc#L221